jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/feedback/SnackbarTests.java

```diff
@@ -1,43 +1,187 @@
 package io.github.epam.material.tests.feedback;
 
+import com.epam.jdi.tools.Timer;
 import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
-import static io.github.com.StaticSite.consecutiveSnackbarFrame;
-import static io.github.com.StaticSite.feedbackSnackbarConsecutivePage;
-import static io.github.com.StaticSite.feedbackSnackbarSimplePage;
-import static io.github.com.StaticSite.simpleSnackbarFrame;
+import java.util.HashMap;
+import java.util.Map;
+
+import static io.github.com.MaterialNavigator.openSection;
+import static io.github.com.pages.feedback.SnackbarPage.*;
+
+/**
+ * To see an example of Snackbar web element please visit https://material-ui.com/components/snackbars/
+ */
 
 public class SnackbarTests extends TestsInit {
 
+    private static Timer timer = new Timer(2000L);
+    private static Map<String, String> map  = new HashMap<String, String>() {{
+        put("topCenter", "TopCenter");
+        put("topRight", "TopRight");
+        put("bottomRight", "BottomRight");
+        put("bottomCenter", "BottomCenter");
+        put("bottomLeft", "BottomLeft");
+        put("topLeft", "TopLeft");
+    }};
+    private static String cssConstanValue = "MuiSnackbar-root MuiSnackbar-anchorOrigin";
+
+    @BeforeMethod
+    public void before() {
+        openSection("Snackbar");
+    }
+
     @Test
     public void simpleSnackbarTest() {
-        feedbackSnackbarSimplePage.open();
 
-        simpleSnackbarFrame.snackbar.is().notExist();
-        simpleSnackbarFrame.openSnackbarButton.click();
-        simpleSnackbarFrame.snackbar.is().visible();
-        simpleSnackbarFrame.snackbar.has().content("Note archived");
-        simpleSnackbarFrame.snackbar.getActionByIndex(1).has().text("UNDO");
-        simpleSnackbarFrame.snackbar.clickActionByIndex(2);
-        simpleSnackbarFrame.snackbar.is().notExist();
+        simpleSnackbarPopUp.is().notVisible();
+        simpleSnackbarButton.click();
+        simpleSnackbarPopUp.is().text("Note archived");
+        undoSnackbarButton.is().text("UNDO");
+        undoSnackbarButton.click();
+        simpleSnackbarPopUp.is().notVisible();
+        simpleSnackbarButton.click();
+        closePopUpSnackbarButton.click();
+        simpleSnackbarPopUp.is().notVisible();
+    }
+
+    @Test
+    public void customizedSnackbarTest() {
+
+        customizedSnackbarPopUpTitles.get(1).is().text("This is an error message!");
+        customizedSnackbarPopUpStyles.get(1).has().classValue(String.format("MuiPaper-root MuiAlert-root MuiAlert-filled%s MuiPaper-elevation6", "Error"));
```

Comment:
Do we really need to check "MuiPaper-root MuiAlert-root MuiAlert-filled%s MuiPaper-elevation6". isn't it enough to check "MuiAlert-filledError"? It looks a bit messy
The same for other lines